Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,46 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  margin: 1em;
+  padding: 3em;
+  position: relative;
+  width: 15%;
```

Comment:
use a fixed pixel value here, if you use it in a really big screen this card would be enormous